CORE-2337 Move info pane controls outside

vladan-m opened the pull request:
Please take a look at this. On page load when you are hovering over info pane controls that are now outside of info pane there is little flickering. But, once you click to change info pane height flickering is gone. How to solve this. I've checked and I couldn't find anything inside CSS, it's like JS is doing something on hover.

Smotko commented:
@vladan-m I looked into it and it's a css issue. If you remove the transition `transition: opacity 0.2s ease;` the problem goes away.

vladan-m commented:
@Smotko If you remove transition: opacity 0.2s then it's not gonna give you difference between hover state and normal state. I've tried that.

Smotko commented:
@vladan-m If I remove this line https://github.com/google/ggrc-core/blob/develop/src/ggrc/assets/stylesheets/_pin.scss#L66 the problem goes away. In any case, it is not a javascript issue.

vladan-m commented:
@Smotko Ohhhh man, what's going on. I was removing transition cause I've assumed that could be a problem but nothing happened. Let me check again. Maybe my compass watch bugged, but it was working nice for everything else. Very strange.

vladan-m commented:
@Smotko Check it now. I've fixed it. Also, dropdown menu for info pane was too mush to the bottom so, I've moved that also to be in one line with title and state.

edofic commented:
Merge conflict.

I see you already had some on this PR but I don't see any merge commits and I don't understand what is going on.
Can you try this approach for solving the current merge conflict?

1. `git fetch upstream` (assuming you have git remote called upstream pointing to `google/ggrc-core`
1. `git merge upstream/develop`
1. use a merge tool of your choice to fix conflicts (like `git mergetool`)
1. `git commit`

This will create a "merge commit" that will preserve history and make what is happening obvious. Feel free to contact me if you have any problems.

vladan-m commented:
@edofic Hey, on first look I would say it's conflict for icons.png file. I will handle that and fix it today. Thanks.

vladan-m commented:
@edofic Merge conflict is fixed. I've also added some minor improvements so that styling for mockups is not in a conflict with real app.
